config/api.yml

```diff
@@ -1068,6 +1068,56 @@
     - :collection
     :verbs: *g
     :klass: ChargebackRateDetailMeasure
+  :middleware_datasources:
+    :description: Middleware Datasources
+    :klass: MiddlewareDatasource
+    :options:
+      - :collection
+    :verbs: *g
```

Comment:
It defines a "set" of accepted methods, GET being index/show, POST being create, PUT/PATCH being update and delete being destroy. Hence, `g`, `gp`, `gd`, `gpd`, `gppd` and so on.